hilti/runtime/src/fiber.cc

```diff
@@ -173,6 +173,33 @@ detail::Fiber::Fiber(Type type) : _type(type), _fiber(std::make_unique<::Fiber>(
     switch ( type ) {
         case Type::Main: {
             ::fiber_init_toplevel(_fiber.get());
+
+#if __x86_64__ || __arm__ || __arm64__ || __aarch64__ || __i386__
```

Comment:
Suggest to move this just around the use of `__builtin_frame_address` since the rest should be generic.